src/envoy/http/jwt_auth/jwt.h

```diff
@@ -231,8 +231,6 @@ class Jwt : public WithStatus {
   int64_t Exp();
 
  private:
-  const EVP_MD* md_;
```

Comment:
I am surprised a pointer can be removed without touching destructor.
Could you elaborate who owns the object `md_` used to reference?

Reply:
Does it matter who owns it? I thought it's a const pointer pointing to a static variable.

Reply:
Talked offline and we think this is not a concern, resolved.

Reply:
Thank you for the explanation! `md_` may not be introduced at the beginning. 
/lgtm

Reply:
Sorry I was heading to a meeting so I didn't complete what I should have pointed out: The `static variable` owns so `Jwt` doesn't need to free it. That's what I wanted to know. Thanks again!